src/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -69,4 +69,14 @@
     <Reference Include="System.Numerics.Vectors" />
     <Reference Include="System.Runtime.CompilerServices.Unsafe" />
   </ItemGroup>
+
+  <!-- The Source Package will call into this target in order to retrieve the compiled resources file as
+       it will need to be part of the source package contents. -->
+  <Target Name="GetSourcesToPackage" Returns="@(_sourceToPackage)" DependsOnTargets="Compile">
+      <ItemGroup>
+        <_sourceToExclude Include="@(Compile)" Condition="$([System.String]::Copy('%(Compile.Identity)').ToLower().EndsWith('assemblyinfo.cs'))" />
+        <_sourceToPackage Include="@(Compile->'%(FullPath)')" PackagePath="contentFiles/cs/netstandard2.0/%(RecursiveDir)%(FileName)%(Extension)" Exclude="@(_sourceToExclude)"/>
```

Comment:
nit: RecursiveDir is always empty here.  If you really wanted a path you might want to run makerelative and use a relative path if available.  Probably not necessary for your usage so consider just omitting RecursiveDir and adding a comment that files need to be uniquely named.